### ISSUE TITLE
rollback to previous version / deploy a specific version

### ISSUE THREAD

**Issue by jtheuer** [boxfuse/boxfuse-issues#75]
How can I either roll back a previous blue-green deployment or deploy a previously built version?
I would need it for example if a new version doesn't behave correctly in production and I want to quickly revert to the last stable one. 

When a rollback is necessary in production, it can be stressful to find out the previous version. I would appreciate if  ```boxfuse run``` logs the version/image it is going to replace.

**Comment by herder** [comment]
Maybe we could have a `boxfuse rollback <optional version>`, which selects the last known good previous version as default?

**Comment by axelfontaine** [comment]
I agree. `boxfuse rollback` could indeed be a very valuable. The optional version doesn't make as much sense, as in that case you may just as well invoke run again.

**Comment by jtheuer** [comment]
Is it already possible with the run command to select an already built image version?

**Comment by axelfontaine** [comment]
@jtheuer Yes, that has been possible from the start. `boxfuse run myapp:1.2.3` will deploy version 1.2.3 of myapp (See the usage example at https://boxfuse.com/docs/commandline/run)

**Comment by jtheuer** [comment]
Oh great, I missed that! (And I had a typo when I tried it with a version):

```
> boxfuse run myapp:54 -env prod
Boxfuse client v.1.18.7.938

ERROR: Unknown environment: true => use one of your available environments: dev,prod,production,test
```

For me, a logging of the current version during bluegreen would be sufficient to roll back. But the idea of an extra command is also interesting.
